src/__tests__/config-loader.test.ts

```diff
@@ -49,38 +44,31 @@ describe("config-loader", (): void => {
     const result = configLoader({
       explicitParams: undefined,
       cwd: "/baz",
-      // eslint-disable-next-line @typescript-eslint/no-explicit-any
-      tsConfigLoader: (_: any) => ({
+      tsConfigLoader: () => ({
         tsConfigPath: "/baz/tsconfig.json",
         baseUrl: "./src",
         paths: {},
       }),
     });
 
     const successResult = result as ConfigLoaderSuccessResult;
-    // assert.equal(successResult.resultType, "success");
-    // assert.equal(successResult.absoluteBaseUrl, join("/baz", "src"));
     expect(successResult.resultType).toBe("success");
     expect(successResult.absoluteBaseUrl).toBe(join("/baz", "src"));
   });
 
-  it("should show an error message when baseUrl is missing", () => {
+  it("should tolerate a missing baseUrl", () => {
```

Comment:
I believe this test was broken by https://github.com/dividab/tsconfig-paths/pull/208.